content/source/docs/enterprise/private/admin/index.html.md

```diff
@@ -17,7 +17,7 @@ This section is about in-application administration, including general settings,
 
 ## Accessing the Admin Interface
 
-Only Private Terraform Enterprise users with the site-admin permission can access the administrative functions.
+Only Private Terraform Enterprise users with the site-admin permission can access the administrative functions. (The first user account in a PTFE instance has the site-admin permission; other users can be given this permission in the "Users" section of the admin pages. See [Promoting a User to Administrator](./resources.html#promoting-a-user-to-administrator) for details.)
```

Comment:
I have mixed feelings about this add, but I can't quite articulate why.

Reply:
Well..... not sure what to do about that. 😅 But, looking at it again... those parentheses don't need to be there, so I'll drop those.